src/test/java/uk/gov/digital/ho/proving/income/audit/AuditClientTest.java

```diff
@@ -106,4 +106,33 @@ public void shouldSetAuditableData() {
         assertThat(auditableData.getEventType()).isEqualTo(INCOME_PROVING_FINANCIAL_STATUS_REQUEST);
         assertThat(auditableData.getData()).isEqualTo("{}");
     }
+
+    @Test
+    public void shouldRetrieveAuditHistory() {
+        List<AuditEventType> eventTypes = Arrays.asList(INCOME_PROVING_FINANCIAL_STATUS_REQUEST, INCOME_PROVING_FINANCIAL_STATUS_RESPONSE);
+        List<AuditRecord> results = new ArrayList<>();
+        ResponseEntity<List<AuditRecord>> resultsEntity = ResponseEntity.ok(results);
+        when(mockRestTemplate.exchange(eq("some history endpoint"), eq(POST), captorHttpEntity.capture(), eq(new ParameterizedTypeReference<List<AuditRecord>>() {}))).thenReturn(resultsEntity);
```

Comment:
As the endpoint names have to match the ones passed into AuditClient in the setUp method for this to work it would be nice if there was a constant or a field equal to "some history endpoint"